symbulate/tests/test_distributions.py

```diff
@@ -17,18 +17,460 @@ def test_p_one(self):
     def test_sum(self):
         X = RV(Bernoulli(.4) ** 5)
         sims = X.apply(sum).sim(Nsim)
+        key_list = [keys for keys in sims.tabulate()]
         obs = list(sims.tabulate().values())
-        exp = Nsim * stats.binom(n=5, p=.4).pmf(range(6))
+        exp = [Nsim * stats.binom(n = 5, p = .4).pmf(k) for k in key_list]
```

Comment:
No spaces around = in arguments to functions. (It should be `n=5, p=.4`.)

Please fix this everywhere below as well.